utility/auth.js

```diff
@@ -51,7 +59,10 @@ const Auth = ({ children }) => {
     >
       {isLoading ? (
         <LoadingScreenContainer>
-          <LoadingDiv>Authenticating...</LoadingDiv>
+          <div>
+            <LoadingImage src={nwPlusReversed} />
+            <LoadingDiv>Authenticating...</LoadingDiv>
```

Comment:
going from landing page to sponsorship will invoke a loading screen for a brief second, but im pretty sure that's because you route back to landing page again?

Reply:
its because in this build sponsorship hasn't been renamed to spocos, but the header component is already setup to route to spocos. What ends up happening is react tries to move to a route that doesn't exist, crashes, and reloads back to the landing screen for some reason